src/backend/Aggregator/Aggregator/ServicesConfiguration/SwaggerConfiguration.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+using Swashbuckle.AspNetCore.Swagger;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+using System.Linq;
+using System.Collections.Generic;
+
+namespace Aggregator.ServicesConfiguration
+{
+    internal static class SwaggerConfiguration
+    {
+        public static void AddSwagger(this IServiceCollection services, IConfiguration configuration)
+        {
+            services.AddSwaggerGen(conf =>
+            {
+                conf.SwaggerDoc(name: "v1", info: new Info()
+                {
+                    Title = "Photo API",
+                    Version = "v1"
+                });
+                conf.AddAuthorization();
+            });
+        }
+
+        private static void AddAuthorization(this SwaggerGenOptions configuration)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'AddAuthorization'.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=5020178)